makeservices/makemysql-real

```diff
@@ -1,153 +1,78 @@
-    MYSQL_ROOT_PW = conf.get('makemysql', 'passwd')
-
-# Tell the user what this script does, and ask if he is ready to continue.
+    mysql_host = conf.get('makemysql', 'host')
+    mysql_root_pw = conf.get('makemysql', 'passwd')
+    return mysql_host, mysql_root_pw
 
 
 def intro_prompt():
-    print()
-    print('This program will create a MySQL database for your account,')
-    print('if one does not already exist. A randomly-generated password')
-    print('will be assigned for access to your database. MySQL passwords')
-    print('are no longer user-selectable because of frequent occurrences of')
-    print('Unix account passwords being compromised when faulty or out-of-date')
-    print('web applications are exploited.')
-    print()
-    print('The randomly-generated password will be displayed on-screen.')
-    print('Please make sure you are in an environment where nobody else will')
-    print('see it when it appears.')
-    print()
-    print("If you are ready to continue, type 'yes'.")
-    print("Typing anything other than 'yes' will abort this script.")
-    print()
-    print('Continue? ')
-    try:
-        inp = sys.stdin.readline().strip()
-        if inp != 'yes':
-            raise None
-    except:
-        print()
-        print('>>> Aborted by user request.')
-        print()
-        sys.exit(0)
-    # Script continues on if user entered 'yes'.
-    print
-
-# Handle the case where the database already exists.
-
-
-def database_already_exists(username):
-    print()
-    print("The MySQL database '%s' already exists." % username)
-    print('The database password will be reset.')
-    print()
-    print('In case you have forgotten how to access your database')
-    print('(or you inherited it, in the case of a group account),')
-    print('an informational email regarding how to access and manage')
-    print('your database is being sent to your OCF email address now.')
-    print('Please check for it momentarily.')
-    print()
-    print('You can also find the same information at:')
-    print()
-    print('  http://ocf.berkeley.edu/docs/services/mysql/')
-    print()
+    print(dedent(
+        """
+        This program will create a MySQL database for your account,
+        if one does not already exist. A randomly-generated password
+        will be assigned for access to your database. MySQL passwords
+        are no longer user-selectable because of frequent occurrences of
+        Unix account passwords being compromised when faulty or out-of-date
+        web applications are exploited.
 
+        The randomly-generated password will be displayed on-screen.
+        Please make sure you are in an environment where nobody else will
+        see it when it appears.
+
+        If you are ready to continue, type 'yes'.
+        Typing anything other than 'yes' will abort this script.
+
+        Continue?
+        """
+    ))
+    inp = sys.stdin.readline().strip()
+    return inp == 'yes'
```

Comment:
nit: normally prefer to use `input('Continue? ') == 'yes'` because then they can use readline keyboard shortcuts. nbd here.